microcosm_eventsource/ddl/proc_events_create.create.ddl

```diff
@@ -0,0 +1,99 @@
+/*
+ * proc_events_create(table_name, events_to_create_table_name, columns)
+ *
+ * Should be only used by data migrations scripts.
+ * Creates events from table table_name - a valid microcosm-event-source event table.
+ * Pass the events to create as row from table events_to_create_table_name.
+ * Must pass model_id_row_name - the row name that links to the parent model
+ * Must pass the listen of expected columns to be inserted into the target event table.
+ * (Example: for company_event table, the model_id_row_name should be company_id)
+ *
+ * Example:
+ *    CREATE TEMP TABLE events_to_create AS (\n"
+ *           SELECT\n"
+ *              '{reassigned_event_id}'::uuid as id,
+ *              extract(epoch from now()) as created_at,
+ *              extract(epoch from now()) as updated_at,
+ *              assignee,
+ *              NULL::timestamp without time zone as deadline,
+ *              task_id,
+ *              'REASSIGNED' as event_type,
+ *              id as parent_id,
+ *              state,
```

Comment:
Using `add_event_type_to_state` and `remove_event_type_to_state` can be handy here